apps/fluent-tester/src/FluentTester.tsx

```diff
@@ -54,8 +53,11 @@ const TestListSeparator = Separator.customize((t) => ({
 }));
 
 export const FluentTester: React.FunctionComponent<FluentTesterProps> = (props: FluentTesterProps) => {
+  // filters for tests specific to each platform
+  const filteredTestComponents = tests.filter((test) => test.platforms.includes(Platform.OS as string));
+
   // sort tests alphabetically by name
-  const sortedTestComponents = props.enabledTests.sort((a, b) => a.name.localeCompare(b.name));
+  const sortedTestComponents = filteredTestComponents.sort((a, b) => a.name.localeCompare(b.name));
```

Comment:
nit: you can combine the comments into a block that's just like "filter and sort tests"